2023/11.md

```diff
@@ -130,7 +130,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
     |   | 3     | 30m | Intl.DurationFormat stage 3 update and normative PRs ([slides](https://notes.igalia.com/p/nxMdcUtbb#/)) | Ben Allen |
     |   | 3     | 30m | :hourglass: Decorators normative update re: extra initializers ([PR](https://github.com/pzuraq/ecma262/pull/12)) | Kristen Hewell Garrett |
     |   | 3     | 30m | Iterator helpers [web compat](https://github.com/tc39/proposal-iterator-helpers/issues/286) continuation (backup [PR](https://github.com/tc39/proposal-iterator-helpers/pull/287)) | Michael Ficarra &amp; Shu-yu Guo |
-    |   | 2     | 15m | [ShadowRealm](https://github.com/tc39/proposal-shadowrealm) stage 2 update | Leo Balter |
+    |   | 2     | 15m | [ShadowRealm](https://github.com/tc39/proposal-shadowrealm) stage 2 update ([slides]([url](https://docs.google.com/presentation/d/17esbBbAlKe1nZzWx47ItpVP1XovIupxKEAL1PgUDFrk/edit#slide=id.g26255f6e788_0_0))) | Leo Balter |
```

Comment:
This is nesting links. Should just be

```
([slides](https://docs.google.com/presentation/d/17esbBbAlKe1nZzWx47ItpVP1XovIupxKEAL1PgUDFrk/edit#slide=id.g26255f6e788_0_0))
```

Reply:
thanks, fixed in a4476e8cd7e077eda9b85a35c07ddbeab732a05d